presto-docs/src/main/sphinx/sql/source.rst

```diff
@@ -0,0 +1,23 @@
+======
+SOURCE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SOURCE file-path
+
+Description
+-----------
+
+Execute all complete sql statements in the given file.
```

Comment:
Actually it works as you run all SQL statements in the file by copy/paste-ing to CLI, so USE works too. Since there is no restriction, I didn't add anything to docs. Do you think I should still state that `USE` works?